### PR TITLE
Docs/install cli #2576

### LINKED ISSUE #2576: Potential improvements in QuickStart documentation - Install lakeFS CLI
A few areas of potential improvements for [https://docs.lakefs.io/quickstart/lakefs_cli.html](https://docs.lakefs.io/quickstart/lakefs_cli.html )

1. The link to Download the CLI binary it takes you to a general download file of "what is lakeFS". It is unclear the way to get it is to get the right version of the tar file and copy the file out of the package (the video actually called that out explicitly).
2. It is worth mentioning you may need to give permissions to the file to execute under your security settings (MacOS will block by default). 
3. The output of the server endpoint URI in the example is "URL: http://localhost:8000/api/v1" while I THINK the default is "URL: http://localhost:8000"
4. Perhaps it makes sense to separate the 3 commands at point 4 and add a brief explanation of what those 3 do.

### PR DESCRIPTION
**Pull request by peacing**
Fix #2576

### CODE REVIEW

**Comment by peacing** [comment]
I didn't test if the Youtube link works. Could use help compiling the docs locally or a reviewer can ideally confirm before merging...

File: docs/quickstart/lakefs_cli.md [hunk 1]
```diff
@@ -12,26 +12,36 @@ has_children: false
...
+  [![IMAGE ALT TEXT](http://img.youtube.com/vi/8nO7RT411nA/0.jpg)](http://www.youtube.com/watch?v=8nO7RT411nA "Install the lakeFS CLI")
+
   Here's how to get started with the CLI:
 
   1. Download the CLI binary:
 
-     [Download lakectl](../index.md#downloads){: .btn .btn-green target="_blank"}
+     The CLI binary can be downloaded from the official [lakeFS releases](https://github.com/treeverse/lakeFS/releases) published on GitHub under "Assets". Unless you need a specific previous version of the CLI, it is recommended to download the most recently released version.
+
+      ![Relase Assets]({{ site.baseurl }}/assets/img/lakefs-release-asset.png)
```
Comment (itaiad200) [inline docs/quickstart/lakefs_cli.md:25]:
```suggestion
      ![Release Assets]({{ site.baseurl }}/assets/img/lakefs-release-asset.png)
```

Comment (itaiad200) [reply docs/quickstart/lakefs_cli.md:25]: Also - did you check that this reference work? (using `make docs-serve`)

**Review by itaiad200 — COMMENTED** [review]
Approved but see my comments.

**Review by itaiad200 — APPROVED** [review]
Approved but see my comments.

**Pull request merged**